### PR TITLE
[ALLUXIO-2620] Fix comment style LRUEvictor.java

### PR DESCRIPTION
**Pull request by ZhouYC627**
ALLUXIO-2620
https://alluxio.atlassian.net/browse/ALLUXIO-2620

### CODE REVIEW

**Comment by ZhouYC627** [comment]
alluxio-bot, check this please

**Comment by calvinjia** [comment]
Jenkins, test this please.

File: core/server/worker/src/main/java/alluxio/worker/block/evictor/LRFUEvictor.java [hunk 1]
```diff
@@ -56,7 +56,7 @@
   private final Map<Long, Double> mBlockIdToCRFValue = new ConcurrentHashMapV8<>();
   // In the range of [0, 1]. Closer to 0, LRFU closer to LFU. Closer to 1, LRFU closer to LRU
   private final double mStepFactor;
-  // In the range of [2, INF]
+  /** The attention factor is in the range of [2, INF] */
```
Comment (calvinjia) [inline core/server/worker/src/main/java/alluxio/worker/block/evictor/LRFUEvictor.java]: `attention` -> `attenuation`

Comment (apc999) [inline core/server/worker/src/main/java/alluxio/worker/block/evictor/LRFUEvictor.java]: A period is required to pass style checker 
So it should look like:
```java
/** The attention factor is in the range of [2, INF]. */
```

Comment (ZhouYC627) [inline core/server/worker/src/main/java/alluxio/worker/block/evictor/LRFUEvictor.java]: I added a new commit, check this please.

**Comment by calvinjia** [comment]
Jenkins, test this please.

**Pull request merged**